qemu/tests/cpu_topology_test.py

```diff
@@ -72,5 +74,8 @@ def check(p_name, exp, check_cmd):
         if not check_if_vm_vcpu_topology_match(session, os_type, vm.cpuinfo):
             test.fail('CPU topology of guest is incorrect.')
         if params.get('check_siblings_cmd'):
-            check('sibling', vcpu_threads * vcpu_cores, 'check_siblings_cmd')
+            check('sibling', vcpu_threads * vcpu_cores, params['check_siblings_cmd'])
+        if params.get('check_core_id_cmd'):
+            for cpu_id in list(range(params['smp'])):
+                check('core_id', cpu_id, params['check_PPTT_cmd'] % cpu_id)
```

Comment:
Also need to be updated.

Reply:
Oh sorry for the mistake.
Corrected

git am 3178.patch 
Applying: cpu_topology_test:Add aarch64 PPTT table check

` (1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.cpu_topology_test.arm64-pci: PASS (64.26 s)`